### DISCUSSION TITLE
Grandia 2 Saves

### DISCUSSION THREAD

**Discussion by tekkaman1** [flyinghead/flycast#568] [Q&A]
Hello,

I've been playing Grandia 2 the PC version but I'm stuck because after about 40 hours of gameplay now it keeps crashing. The developers said that it's based on the Dreamcast version and that it's possible to transfer the save files back and forth from Dreamcast to the PC version. I'm tired of the game crashing on PC so I would like to play it using your emulator. But I don't know how to transfer the save files as they seem a little different. The save files from the game say GRANDIA2_000.VMS and GRANDIA2_001.VMS.

**Comment by flyinghead** [comment]
I believe there are free tools to import and export .VMS files to/from VMU save files. 
See https://segaretro.org/VMU_Explorer
